CHANGELOG.md

```diff
@@ -13,6 +13,8 @@
 #### Bug fixes
 
 - Properly specify configuration statuses for RUM and serverless.
+- Clarify requirement for RUM `beaconEndpoint` configuration value to be HTTPS
+  only on systems that allow such enforcement.
```

Comment:
So...to clarify a bit... In the Android library, we *could* add an additional check that required the user to set things to https, and allow that to be overridden by a the config option. But, it wouldn't actually do anything useful, since whether to allow bare `http` client requests (or not, which is the default behavior) is controlled by the Android app manifest itself.

Reply:
I'm a bit out of my depth on this one. Does this mean we need to change the language of this PR to better address the situation?

Reply:
no, I think the wording there is fine. I guess I just wanted to highlight the nuance that we *could* enforce that the endpoint that people provided was https, but it wouldn't really have any meaningful impact. Or, perhaps more importantly, if we gave them an override, it would be misleading, since they would still have to configure their app to use insecure http calls for it to do anything.